igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerMonitor.groovy

```diff
@@ -187,6 +192,27 @@ class DockerMonitor extends CommonPollingMonitor<ImageDelta, DockerPollingDelta>
             account: image.account,
           ), artifact: dockerArtifact))
         }
+
+        if (keelService.isPresent()) {
+          String imageReference = image.repository + ":" + image.tag
+          Artifact artifact = new Artifact(
```

Comment:
I'd encourage using `Artifact.Builder` here instead of calling the all-args constructor; otherwise any additional (optional) fields that get added to artifact in the future will break this call when kork is bumped.  (Maybe we should even make that constructor private assuming it hasn't been used too many places already.)

Reply:
Oh yes that will be much better. Overlooked the builder. Will change! Thanks.

Reply:
I think it would be worth making that constructor private